data/102/063/191/102063191.geojson

```diff
@@ -10,6 +10,12 @@
     "geom:latitude":47.605167,
     "geom:longitude":9.882882,
     "iso:country":"DE",
+    "label:deu_x_preferred_longname":[
+        "Landkreis Lindau (Bodensee)"
```

Comment:
` (Bodensee)` is a disambiguation name. Please fix up the names so express that, and strip here.